internal/app/go-filecoin/node/test/builder.go

```diff
@@ -115,3 +120,37 @@ func (b *NodeBuilder) requireNoError(err error) {
 	b.tb.Helper()
 	require.NoError(b.tb, err)
 }
+
+// MakeNodeWithChainSeed makes a single node with the given chain seed, and some init options
+func MakeNodeWithChainSeed(t *testing.T, seed *node.ChainSeed, builderopts []node.BuilderOpt, initopts ...node.InitOpt) *node.Node { // nolint: golint
+	t.Helper()
+	builder := NewNodeBuilder(t)
+	builder.WithGenesisInit(seed.GenesisInitFunc)
+	builder.WithBuilderOpt(builderopts...)
+	builder.WithInitOpt(initopts...)
+	return builder.Build(context.Background())
+}
+
+// MakeNodesUnstartedWithGif creates some new nodes with an InMemoryRepo and fake proof verifier.
+// The repo is initialized with a supplied genesis init function.
+// Call StartNodes to start them.
+func MakeNodesUnstartedWithGif(t *testing.T, numNodes int, builderopts []node.BuilderOpt, gif consensus.GenesisInitFunc) []*node.Node {
+	builder := NewNodeBuilder(t)
+	builder.WithGenesisInit(gif)
+	builderopts = append(builderopts, node.DefaultTestingConfig()...)
+	builder.WithBuilderOpt(builderopts...)
+
+	var out []*node.Node
+	for i := 0; i < numNodes; i++ {
+		nd := builder.Build(context.Background())
+		out = append(out, nd)
+	}
```

Comment:
Move this loop into a `builder.BuildMany(numNodes)` and then my same comment as above applies about removing this function and using the builder directly.

"Unstarted" is redundant in the name - we don't have anything that makes them already started.